[2025-02-18T04:32:35.917+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:32:35.934+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-12-27T00:00:00+00:00 [queued]>
[2025-02-18T04:32:35.945+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-12-27T00:00:00+00:00 [queued]>
[2025-02-18T04:32:35.946+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:32:35.959+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-12-27 00:00:00+00:00
[2025-02-18T04:32:35.968+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-12-27T00:00:00+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpa2vnd75k']
[2025-02-18T04:32:35.970+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1196) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:32:35.972+0000] {standard_task_runner.py:105} INFO - Job 316: Subtask hello_task
[2025-02-18T04:32:35.974+0000] {standard_task_runner.py:72} INFO - Started process 1204 to run task
[2025-02-18T04:32:35.992+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:36.041+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-12-27T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:32:36.043+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:36.136+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-12-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-27T00:00:00+00:00'
[2025-02-18T04:32:36.138+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:32:36.157+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:32:36.158+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:32:36.173+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:32:36.174+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-12-27T00:00:00+00:00, execution_date=20241227T000000, start_date=20250218T043235, end_date=20250218T043236
[2025-02-18T04:32:36.245+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:32:36.260+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
